.lintstagedrc.js

```diff
@@ -3,12 +3,6 @@ module.exports = {
   '*': ['eclint fix', 'git add'],
   '*.js': ['eslint --fix', 'prettier --write', 'git add'],
   '*.md': ['prettier --write', 'markdownlint', 'git add'],
-  'yarn.lock': ['node ./scripts/yarn-lock-scan.js'],
-  '*package.json': absolutePaths => {
-    const sortPackages = [];
-    absolutePaths.forEach(p => {
-      sortPackages.push(`node ./scripts/sort-package-json.js ${p}`);
-    });
-    return [...sortPackages, `node ./scripts/lint-versions.js`];
```

Comment:
linting still makes sense?

Reply:
I don't think so right? Because `changeset version` already ensures that interdependent packages are kept updated to latest (or if incompatible, it patches them).

Reply:
still makes sense for new packages getting out of sync and root dev deps